bot/bot.go

```diff
@@ -5,24 +5,25 @@ import (
 
 	log "github.com/sirupsen/logrus"
 
+	"github.com/jaimeteb/chatto/channels"
 	"github.com/jaimeteb/chatto/clf"
-	cmn "github.com/jaimeteb/chatto/common"
-	"github.com/jaimeteb/chatto/ext"
+	"github.com/jaimeteb/chatto/extension"
 	"github.com/jaimeteb/chatto/fsm"
+	"github.com/jaimeteb/chatto/query"
 	"github.com/spf13/viper"
 )
 
 // Bot models a bot with a Classifier and an FSM
 type Bot struct {
 	Name       string
 	Machines   fsm.StoreFSM
-	Domain     fsm.Domain
+	DB         *fsm.DB
 	Classifier clf.Classifier
-	Extension  ext.Extension
-	Clients    Clients
+	Extension  extension.Extension
+	Channels   *channels.Channels
```

Comment:
Renamed `Clients` to `Channels` because that's what we are using. Seems to make more sense.